packages/node-sdk/src/client.ts

```diff
@@ -583,6 +626,12 @@ export class BucketClient {
   }
 
   private _getFeatures(context: Context): Record<string, RawFeature> {
```

Comment:
something like this would simplify:
```suggestion
  private _getFeatures({enableTracking=true, ...context}: Context & {enableTracking: boolean}): Record<string, RawFeature> {
```

Reply:
Good one